<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dlg_AddDriver</class>
 <widget class="QDialog" name="Dlg_AddDriver">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>409</width>
    <height>397</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>车辆进场信息</string>
  </property>
  <widget class="QWidget" name="widget" native="true">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>330</y>
     <width>261</width>
     <height>30</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <property name="spacing">
     <number>0</number>
    </property>
    <item row="0" column="0">
     <widget class="QPushButton" name="Btn_save">
      <property name="text">
       <string>确定</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QPushButton" name="Btn_cancel">
      <property name="text">
       <string>取消</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>24</y>
     <width>311</width>
     <height>311</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout_2">
    <item row="0" column="0">
     <widget class="QLabel" name="label_name">
      <property name="minimumSize">
       <size>
        <width>80</width>
        <height>0</height>
       </size>
      </property>
      <property name="text">
       <string>姓名</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QLineEdit" name="line_name"/>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="label_carColor">
      <property name="minimumSize">
       <size>
        <width>80</width>
        <height>0</height>
       </size>
      </property>
      <property name="text">
       <string>车主编号</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QLineEdit" name="line_dno"/>
    </item>
    <item row="2" column="0">
     <widget class="QLabel" name="label_carColor_2">
      <property name="minimumSize">
       <size>
        <width>80</width>
        <height>0</height>
       </size>
      </property>
      <property name="text">
       <string>车编号</string>
      </property>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="QLineEdit" name="line_cno"/>
    </item>
    <item row="3" column="0">
     <widget class="QLabel" name="label_licenseNum">
      <property name="minimumSize">
       <size>
        <width>80</width>
        <height>0</height>
       </size>
      </property>
      <property name="text">
       <string>车牌号</string>
      </property>
     </widget>
    </item>
    <item row="3" column="1">
     <widget class="QLineEdit" name="line_clicense"/>
    </item>
    <item row="4" column="0">
     <widget class="QLabel" name="label_seatNum">
      <property name="minimumSize">
       <size>
        <width>80</width>
        <height>0</height>
       </size>
      </property>
      <property name="text">
       <string>停车位编号</string>
      </property>
     </widget>
    </item>
    <item row="4" column="1">
     <widget class="QComboBox" name="comboBox_pno"/>
    </item>
    <item row="5" column="0">
     <widget class="QLabel" name="label_seatPos">
      <property name="minimumSize">
       <size>
        <width>80</width>
        <height>0</height>
       </size>
      </property>
      <property name="text">
       <string>车辆类型</string>
      </property>
     </widget>
    </item>
    <item row="5" column="1">
     <widget class="QComboBox" name="comboBox_cartype">
      <item>
       <property name="text">
        <string>月卡</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>临时</string>
       </property>
      </item>
     </widget>
    </item>
    <item row="6" column="0">
     <widget class="QLabel" name="label_seatPrice">
      <property name="minimumSize">
       <size>
        <width>80</width>
        <height>0</height>
       </size>
      </property>
      <property name="text">
       <string>进场时间（小时）</string>
      </property>
     </widget>
    </item>
    <item row="6" column="1">
     <widget class="QLineEdit" name="line_cinHour"/>
    </item>
    <item row="7" column="0">
     <widget class="QLabel" name="label_departTimeHour">
      <property name="minimumSize">
       <size>
        <width>80</width>
        <height>0</height>
       </size>
      </property>
      <property name="text">
       <string>进场时间（分钟）</string>
      </property>
     </widget>
    </item>
    <item row="7" column="1">
     <widget class="QLineEdit" name="line_cinMin"/>
    </item>
    <item row="8" column="0">
     <widget class="QLabel" name="label_departTimeHour_2">
      <property name="minimumSize">
       <size>
        <width>80</width>
        <height>0</height>
       </size>
      </property>
      <property name="text">
       <string>保安号</string>
      </property>
     </widget>
    </item>
    <item row="8" column="1">
     <widget class="QLineEdit" name="line_username"/>
    </item>
   </layout>
  </widget>
 </widget>
 <tabstops>
  <tabstop>line_name</tabstop>
  <tabstop>line_dno</tabstop>
  <tabstop>line_clicense</tabstop>
  <tabstop>line_cinHour</tabstop>
  <tabstop>line_cinMin</tabstop>
  <tabstop>comboBox_cartype</tabstop>
  <tabstop>Btn_save</tabstop>
  <tabstop>Btn_cancel</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
